src/sass-inject.js

```diff
@@ -60,9 +62,13 @@ const compile = scss => {
 
 export default load => {
   urlBase = load.address;
+  let indentedSyntax = urlBase.split('.').slice(-1)[0] == 'sass'
```

Comment:
You can use the shiny new [String.prototype.endsWith()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/endsWith) method here.